packages/wonder-blocks-core/src/hooks/__tests__/use-is-mounted.test.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import {renderHook} from "@testing-library/react-hooks";
+
+import {useIsMounted} from "../use-is-mounted.js";
+
+describe("useIsMounted", () => {
+    it("should return false on first call", () => {
```

Comment:
**suggestion:** The Arrange/Act/Assert comments to divide sections are missing here.

See best practices: https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/98402353/JavaScript+Testing+Best+Practices#Arrange-Act-Assert